sections/03_categorize.md

```diff
@@ -63,6 +63,29 @@ subsequent training. Similar strategies have been employed for EHR data where
 high-quality labeled examples are also difficult to obtain
 [@doi:10.1101/039800].
 
+Chest X-ray image analysis is another area where recent DL methods showed
+remarkable results [@arXiv:1701.06599 @arXiv:XXX]. Alternative to using fully
+labeled training data, the authors in [@arXiv:XXX] proposed to use weakly
+labeled images for training deep learning models. To generate weak labels for
+x-ray images, Wang et al. [1] applied a series of Natural Language Processing
+(NLP) techniques to the associated chest X-ray radiological reports.
+Specifically, they first extracted all diseases mentioned in the reports using a
+state-of-the-art NLP tool, then applied a newly developed negation and
+uncertainty detection tool (NegBio) to filter negative and equivocal findings in
+the reports. Evaluation on three independent datasets demonstrated that NegBio
+is highly accurate for detecting negative and equivocal findings (~90% in
+F-measure overall). These highly accurate results meet the need to generate a
+corpus with weak labels, which serves as a solid foundation for the later
+process of image classification. The resulting dataset, known as CXR-XIV,
```

Comment:
Some of the by-the-by detail could be excised, like ", known as CXR-XIV,". Makes it shorter and easier to read.